src/main/java/ai/rev/speechtotext/ApiClient.java

```diff
@@ -194,6 +195,7 @@ public String getTranscriptText(String id) throws IOException {
    * @see <a
    *     href="https://docs.rev.ai/api/asynchronous/reference/#operation/SubmitTranscriptionJob">https://docs.rev.ai/api/asynchronous/reference/#operation/SubmitTranscriptionJob</a>
    */
+  @Deprecated
```

Comment:
Do we have to deprecate this method? It is a convenience method for submitting a job using the url.

Reply:
We wouldn't have to I guess, but it sets the media url property which is already deprecated. That method is still supported. If they want to add both a url and options, I think pointing them to just add the source config to the options makes more sense.

Reply:
I was thinking that we can keep the method and change the internals to use source_config instead of media_url property.

This would mean that customers will automatically start using source config. They can use the options method in case they want to provide the more advanced auth header.

Reply:
I do see your point that it might be better to deprecate this method. It makes the methods simpler